configure.ac

```diff
@@ -344,7 +344,7 @@ AC_CHECK_HEADERS([boost/lexical_cast.hpp boost/functional/hash.hpp],
                 [AC_MSG_ERROR([boost is not installed.])])
 
 AC_CHECK_HEADER([picojson.h],
```

Comment:
Checking for `picojson.h` directly seems pretty wrong since if we by some coincidence would pick up a `picojson.h` from the system we would actually be required to define `PICOJSON_USE_INT64` ourself. OTOH, this header is never included directly here, but instead only via stout.

I would remove the check for `picojson.h` here and instead rely on the Mesos installation to provide it.